File: 1291-sequential-digits/1291-sequential-digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        vector<int> res;
        for(char i = 1; i < 10; i++)
        {
            char cur = i;
            int curNum = i;
            while(curNum <= high && cur <= 9)
            {
                if(curNum >= low)
                {
                    res.push_back(curNum);
                }
                cur++;
                curNum = curNum * 10 + cur;
            }
        }
        sort(res.begin(), res.end());
        return res;
    }
};

File: 1291-sequential-digits/1291-sequential-digits.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        res = []
        for i in range(1, 10):
            cur = i 
            curNum = cur
            while curNum <= high and cur <= 9:
                if curNum >= low:
                    res.append(curNum)
                cur += 1
                curNum = curNum * 10 + cur
        res.sort()
        return res